Sweeper/ViewControllers/PinDetailsViewController.swift

```diff
@@ -16,17 +15,17 @@ class PinDetailsViewController: UIViewController {
     @IBOutlet weak var messageLabel: UILabel!
     @IBOutlet weak var messageImage: UIImageView!
     
-    var pinMarker: PinMarker!
+    var pinAnnotation: PinAnnotation!
 
     override func viewDidLoad() {
         super.viewDidLoad()
 
 //         pinMarker.pin.markAsViewed(User.currentUser)
```

Comment:
Need to change this line as well, but if we're going with the tour idea, then this line won't need any more